hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/metrics/HybridSubscriptionMetricsRepository.java

```diff
@@ -10,44 +10,26 @@
 import pl.allegro.tech.hermes.infrastructure.zookeeper.ZookeeperPaths;
 import pl.allegro.tech.hermes.management.domain.subscription.SubscriptionLagSource;
 import pl.allegro.tech.hermes.management.domain.subscription.SubscriptionMetricsRepository;
-import pl.allegro.tech.hermes.management.infrastructure.graphite.GraphiteClient;
-import pl.allegro.tech.hermes.management.infrastructure.graphite.GraphiteMetrics;
-import pl.allegro.tech.hermes.management.stub.MetricsPaths;
+import pl.allegro.tech.hermes.management.infrastructure.metrics.MonitoringSubscriptionMetricsProvider.MonitoringSubscriptionMetrics;
 
 import java.util.function.Supplier;
 
 import static org.apache.commons.lang.exception.ExceptionUtils.getRootCauseMessage;
-import static pl.allegro.tech.hermes.common.metric.HermesMetrics.escapeDots;
 
 @Component
```

Comment:
It would be nice to be able to disable the whole monitoring, but we can do that in the future